src/docs/product/integrations/msteams/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Microsoft Teams
+sidebar_order: 1
+---
+
+Microsoft Teams is a hub for teamwork in Office 365. Keep all your team's chats, meetings, files, and apps together in one place. Get alerts that let you assign, ignore, and resolve issues right in your Teams channels with the Sentry integration for Microsoft Teams. 
+
+This integration needs to set up only once per organization, then it is available for _all_ projects.
+
+## Install
+
+1. Visit <https://teams.microsoft.com/l/app/5adee720-30de-4006-a342-d454317db1d4>
```

Comment:
Is this what the final link will look like or will it be updated? I think it'd look cleaner if it were a clickable piece of text instead of this kinda ugly URL.